packages/config/categories.ts

```diff
@@ -25,19 +25,14 @@ export const categories: CategoryDetails = {
 		name: 'CSS',
 		description: 'Can be used to style your components',
 	},
-	tools: {
-		id: 'tools',
-		name: 'Tools',
-		description: 'Different tools',
+	additionalFunctionality: {
+		id: 'additionalFunctionality',
```

Comment:
we could make the id be just `additional` to keep it shorter

Reply:
Good suggestion!